openapi/components/parameters/communityDistrictIdQueryParam.yaml

```diff
@@ -0,0 +1,10 @@
+name: communityDistrictId
+required: false
+in: query
+schema:
+  type: string
+  pattern: ^([0-9]{3})$
+  example: '101'
+  nullable: true
```

Comment:
This shouldn't be `nullable`. Removing this line will change it to the default value of `false`. Having `null` as a default value in the service is fine, but the actual query param isn't `nullable`.

Reply:
This conflicts with Tim's request [above](https://github.com/NYCPlanning/ae-zoning-api/pull/411#discussion_r1941383063).

Changing this changes the `FindCapitalProjectsQueryParams` type to disallow the option for `communityDistrictId`  from being `null`, which then prevents us from using `null` to represent the lack of `communityDistrictId`.